### ISSUE TITLE
CREATE TABLE + WITH clause

### ISSUE THREAD

**Issue by ImreSamu** [darold/pgFormatter#120]
Thank you for this project!

Using (`pg_format version 4.0`), I have a strange results with the following example:

#### input with optimal formatting :

```sql
CREATE TABLE _temp_stats AS 
WITH _table_stats AS (
    SELECT
        COUNT(table_name) AS tables_n,
        COUNT(table_name) FILTER (WHERE table_schema = 'public') AS public_tables_n
    FROM
        information_schema.tables
)
SELECT 
   * 
FROM 
   _table_stats
;

```

#### output:

```sql
CREATE TABLE _temp_stats AS WITH _table_stats AS (
    SELECT
        COUNT(table_name
) AS tables_n,
        COUNT(table_name) FILTER (WHERE table_schema = 'public'
) AS public_tables_n FROM information_schema.tables
)
SELECT
    * FROM _table_stats;

```

imho:   if possible,  the `output` should be similar than the `input`

**Comment by darold** [comment]
Yes this is one of the remaining formatting that fail with nested CTE. I will work on it this summer.

**Comment by darold** [comment]
Actually it is fixed by commit b3d0208.

**Comment by ImreSamu** [comment]
Thank you!
I can confirm - it is OK :heart: 
 
current results: 

```sql
CREATE TABLE _temp_stats AS WITH _table_stats AS (
    SELECT
        COUNT(table_name) AS tables_n,
        COUNT(table_name) FILTER (WHERE table_schema = 'public') AS public_tables_n
    FROM
        information_schema.tables
)
SELECT
    *
FROM
    _table_stats;
```
